Makefile

```diff
@@ -13,11 +13,18 @@
 # limitations under the License.
 
 CMDS=nfsplugin
-all: build
```

Comment:
what does this mean?

Reply:
fixed it. I meant to add an extra command: `tests`